src/main/perl/Path.pm

```diff
@@ -544,6 +559,99 @@ sub directory
 }
 
 
+=item _make_link
+
+This method is mainly a wrapper over C<LC::Check::link>
+returning the standard CAF::Path return values. Every option
+supported by C<LC::Check::link> is supported. C<NoActionFlag>
+is handled by C<LC::Check::link>.
+
+This in internal method, not supposed to be called directly.
+Either call C<symlink> or C<hardlink> public methods instead.
+
+=cut
+
+sub _make_link
+{
+    my ($self, $path, $target, %opts) = @_;
+    $path = $self->_untaint_path($path, "symlink") || return;
+
+    $self->_reset_exception_fail('symlink');
+
+    my $status = LC::Check::link($path, $target, %opts);
```

Comment:
use the wrapper to call `LC`, eg https://github.com/quattor/CAF/blob/master/src/main/perl/Path.pm#L532